pkg/tekton/metapipeline/metapipeline_test.go

```diff
@@ -17,6 +17,7 @@ import (
 	. "github.com/onsi/gomega"
 
 	"github.com/jenkins-x/jx/pkg/client/clientset/versioned/fake"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Can we also add a bdd test before we close the issue please.

 Currently this PR is set to close the issue so to resolve my review just remove the fixes and make it a reference.

Reply:
> Currently this PR is set to close the issue so to resolve my review just remove the fixes and make it a reference.

So you want me to change the commit message so that GitHub won't close the issue when this gets merged, otherwise, we are good?

Reply:
Right.